server/service/src/invoice/prescription/insert/generate.rs

```diff
@@ -40,7 +40,7 @@ pub fn generate(
         comment: None,
         their_reference: None,
         transport_reference: None,
-        allocated_datetime: Some(current_datetime),
+        allocated_datetime: None,
```

Comment:
Removed this default, so allocated_datetime won't be set now, unless backdating...